nautobot/extras/api/customfields.py

```diff
@@ -42,7 +31,7 @@ def __call__(self, serializer_field):
         # Populate the default value for each CustomField
         value = {}
         for field in fields:
-            key = field.slug if use_slug else field.name
+            key = field.name
```

Comment:
Does this need to wait for #824 before switching it back to slug?

Reply:
Yes. I initially changed this as `field.slug` here but that made things work incorrectly - need to use `name` here because that's what is still used in `<model>._custom_field_data[...]`